src/evaluate.cpp

```diff
@@ -65,8 +65,8 @@ Value Eval::evaluate(const Eval::NNUE::Networks&    networks,
 
     Value nnue = (125 * psqt + 131 * positional) / 128;
 
-    // Re-evaluate the position when higher eval accuracy is worth the time spent
-    if (smallNet && (nnue * psqt < 0 || std::abs(nnue) < 227))
+    // Re-evaluate with the big-net if the small-net's NNUE evaluation is below a certain threshold.
```

Comment:
i mean this comment is just useless, with your change it explains the condition in english... the previous comment actually explained why we do this..

Reply:
Comment change reverted

Reply:
I will re-open the PR to fix the check error

Reply:
no...